core/services/feeds/service_test.go

```diff
@@ -32,7 +32,7 @@ import (
 	"github.com/smartcontractkit/chainlink/v2/core/services/chainlink"
 	"github.com/smartcontractkit/chainlink/v2/core/services/feeds"
 	"github.com/smartcontractkit/chainlink/v2/core/services/feeds/mocks"
-	"github.com/smartcontractkit/chainlink/v2/core/services/feeds/proto"
+	proto "github.com/smartcontractkit/chainlink/v2/core/services/feeds/proto"
```

Comment:
this change is not needed?

Reply:
It is required because the pacakge name of the generated proto file has changed

Reply:
the package name was protos and is now named feedsmanager. SO I guess it's either this or everything renamed to feedsmanager

Reply:
got it!